textdb/textdb-perftest/src/main/java/edu/uci/ics/textdb/perftest/excelsink/ExcelSinkTest.java

```diff
@@ -0,0 +1,134 @@
+package edu.uci.ics.textdb.perftest.excelsink;
```

Comment:
Thanks for all the comments! I have modified the code based on @zuozhiw 's comments. 
For the test cases, I can add 1. tests on different field types; 2. tests on null or empty types or even Chinese characters; 3. tests on large number of tuples. Is there any other cases you would also recommend testing? @chenlica